test/controllers/merchants_controller_test.rb

```diff
@@ -74,14 +74,14 @@
 
     it "merchants can access their own account page" do
         login(merchant, :github)
-        flash[:success].must_equal "sappy1 is logged in!"
+        flash[:result_text].must_equal "You successfully logged in as sappy1."
         get merchant_path(merchant.id)
         must_respond_with :success
     end
 
     it "merchants can't access other merchants account page" do
         login(merchant, :github)
-        flash[:success].must_equal "sappy1 is logged in!"
+        flash[:result_text].must_equal "You successfully logged in as sappy1."
```

Comment:
make tests dynamic in case fixtures change

flash[:result_text].must_equal "You successfully logged in as #{merchant.username}."